### PR TITLE
Scala3doc: fix issues with loading stdlib

### PR DESCRIPTION
**Pull request by abgruszecki**

### CODE REVIEW

File: scala3doc/src/dotty/dokka/tasty/comments/MemberLookup.scala [hunk 1]
```diff
@@ -59,7 +59,7 @@ trait MemberLookup {
     import dotty.tools.dotc
     given dotc.core.Contexts.Context = rootContext.asInstanceOf
     val sym = rsym.asInstanceOf[dotc.core.Symbols.Symbol]
-    val members = sym.info.decls.iterator
+    val members = sym.info.decls.iterator.filter(_.denot.isAbsent(false))
```
Comment (smarter) [inline scala3doc/src/dotty/dokka/tasty/comments/MemberLookup.scala]: Should be `isAbsent(canForce = true)` if you don't want false positives.

Comment (abgruszecki) [reply scala3doc/src/dotty/dokka/tasty/comments/MemberLookup.scala]: Interestingly enough `canForce = true` results in exceptions, but `canForce = false` doesn't result in false positives in cases we care about.

File: project/Build.scala [hunk 2]
```diff
@@ -1506,15 +1508,24 @@ object Build {
               (`scala3-interfaces`/Compile/products).value,
               (`tasty-core-bootstrapped`/Compile/products).value,
               (`scala3-library-bootstrapped`/Compile/products).value,
-              // TODO we can't load stdlib from Tasty
```
Comment (romanowski) [inline project/Build.scala:1509]: IMHO we should leave this here until we will be able to generate stdlib togeter with rest of Scala3 jars

File: project/Build.scala [hunk 3]
```diff
@@ -1506,15 +1508,24 @@ object Build {
...
             ).flatten
 
             val roots = joinProducts(dottyJars)
 
             if (dottyJars.isEmpty) Def.task { streams.value.log.error("Dotty lib wasn't found") }
             else generateDocumentation(roots, "Scala 3", "scala3", "-p scala3doc/scala3-docs --projectLogo scala3doc/scala3-docs/logo.svg  --revision master")
           }.value,
+
+          generateScala3StdlibDocumentation:= Def.taskDyn {
+            val dottyJars: Seq[java.io.File] = Seq(
```
Comment (romanowski) [inline project/Build.scala:1520]: it should be stdLibJars

File: project/Build.scala (hunk 2 above)
Comment (abgruszecki) [reply project/Build.scala:1509]: We have a similar comment above, where we define `generateScala3StdlibDocumentation`.

**Pull request merged**